ocrd-olena-binarize

```diff
@@ -24,7 +24,8 @@ set -o pipefail
 # by inserting an AlternativeImage to the PAGE-XML.
 # Regardless, adds new image and new PAGE-XML to METS.
 
-which ocrd >/dev/null 2>/dev/null || { echo "ocrd not in \$PATH. Panicking"; exit 1; }
+which ocrd >/dev/null 2>/dev/null || { echo >&2 "ocrd not in \$PATH. Panicking"; exit 1; }
+((BASH_VERSINFO[0]<4 || BASH_VERSINFO[1]<4)) && echo >&2 "bash $BASH_VERSION is too old. Panicking" && exit 1
```

Comment:
Travis fails here:

    bash 4.3.48(1)-release is too old. Panicking

What do you want to check? I'm afraid that even a recent bash fails now:

    $ ((BASH_VERSINFO[0]<4 || BASH_VERSINFO[1]<4)) && echo >&2 "bash $BASH_VERSION is too old. Panicking"
    bash 5.0.11(1)-release is too old. Panicking

Reply:
```suggestion
((BASH_VERSINFO<4 || BASH_VERSINFO==4 && BASH_VERSINFO[1]<4)) && echo >&2 "bash $BASH_VERSION is too old. Please install bash 4.4 or newer." && exit 1
```

Reply:
> Travis fails here:
> 
> ```
> bash 4.3.48(1)-release is too old. Panicking
> ```

I'm afraid we'd have to abandon this Travis configuration. 4.4 is a sane minimum (see disussion in the issue)